services/QuillLMS/spec/controllers/api/v1/concept_feedback_controller_spec.rb

```diff
@@ -27,6 +26,89 @@
           .reduce({}) { |agg, q| agg.update({q.uid => q.as_json}) }
       )
     end
+
+  end
+
+  describe "#translations" do
+    let(:locale) { Translatable::DEFAULT_LOCALE }
+    let!(:concept_feedback_untranslated) { create(:concept_feedback)}
+    let(:cache_key) {
+      "#{ConceptFeedback::ALL_CONCEPT_FEEDBACKS_KEY}_#{concept_feedback_untranslated.activity_type}_#{locale}"
+    }
+
+    context 'there is a translation available for the language' do
+      let!(:concept_feedback1) { create(:concept_feedback, :with_translated_text)}
+      let!(:concept_feedback2) { create(:concept_feedback, :with_translated_text)}
+
+      context 'there is no cache set' do
+        before do
+          $redis.del(cache_key)
+        end
+
+        it 'returns all the concept feedback translated fields for that language' do
+          get :translations, params: {
+              locale:,
+              activity_type: concept_feedback1.activity_type
+          }, as: :json
```

Comment:
Take or leave this, but since this `get` call is repeated in both `it` blocks, I think you might just want to put the `get` call in the `before` block in this context.